pkg/controller/miganalytic/volume_adjustment.go

```diff
@@ -63,7 +63,14 @@ const (
 	BinarySIGiga  = DFBaseUnit("G")
 )
 
-// DFCommand represents a df command
+// Proposed volume size computation reasons
+const (
+	VolumeAdjustmentNoOp             = "PV sizes not altered. Using original size"
+	VolumeAdjustmentUsageExceeded    = "PV size was adjusted by adding extra headroom of 3%"
+	VolumeAdjustmentCapacityMismatch = "Provisioned size not equal to the requested size, using provisioned size"
```

Comment:
```suggestion
	VolumeAdjustmentNoOp             = "No change in original PV capacity is needed."
	VolumeAdjustmentUsageExceeded    = "PV usage is close to 100%, an extra headroom is added to avoid disk capacity issue in the target cluster."
	VolumeAdjustmentCapacityMismatch = "Requested capacity of PV is not equal to its actual provisioned capacity.  Maximum of both values is used to avoid disk capacity issue in the target cluster."
```

Reply:
Done